### PR TITLE
Sync RUM and CR app configs with Partner center

### PR DESCRIPTION
**Pull request by crystalqdm**
This PR is to sync the app config files with the Partner Center.

For future, 
To link the Partner Centre app details with the app config details, run `shopify app config link` in the app's root. 

This will update your local config file with the details configured in the Partner center.

If you update anything in your local config file, run `shopify app config push` to push your changes to the Partner Center

### CODE REVIEW

File: crash-reporting/shopify.app.toml [hunk 1]
```diff
@@ -2,18 +2,25 @@
...
+  "https://marketplace.raygun.com/external/shopify/redirect/crashreporting"
+]
 
 [webhooks]
 api_version = "2023-10"
 
+  [webhooks.privacy_compliance]
+  customer_deletion_url = "https://app.raygun.com"
+  customer_data_request_url = "https://app.raygun.com"
+  shop_deletion_url = "https://app.raygun.com"
```
Comment (JasenPalmer) [inline crash-reporting/shopify.app.toml]:
```suggestion
  customer_deletion_url = "https://marketplace.raygun.com/shopify/webhook/crashreporting/customer/redact"
  customer_data_request_url = "https://marketplace.raygun.com/shopify/webhook/crashreporting/customer"
  shop_deletion_url = "https://marketplace.raygun.com/shopify/webhook/crashreporting/shop/redact"
```

File: real-user-monitoring/shopify.app.toml [hunk 2]
```diff
@@ -2,18 +2,25 @@
...
+  "https://marketplace.raygun.com/external/shopify/redirect/rum"
+]
 
 [webhooks]
 api_version = "2023-10"
 
+  [webhooks.privacy_compliance]
+  customer_deletion_url = "https://app.raygun.com"
+  customer_data_request_url = "https://app.raygun.com"
+  shop_deletion_url = "https://app.raygun.com"
```
Comment (JasenPalmer) [inline real-user-monitoring/shopify.app.toml]:
```suggestion
  customer_deletion_url = "https://marketplace.raygun.com/shopify/webhook/rum/customer/redact"
  customer_data_request_url = "https://marketplace.raygun.com/shopify/webhook/rum/customer"
  shop_deletion_url = "https://marketplace.raygun.com/shopify/webhook/rum/shop/redact"
```

**Review by JasenPalmer — COMMENTED** [review]
Looks good. Just added suggestions for the webhook urls

File: crash-reporting/shopify.app.toml (hunk 1 above)
Comment (crystalqdm) [reply crash-reporting/shopify.app.toml]: completely missed those, thanks!

**Pull request merged**